app/build.gradle.kts

```diff
@@ -4,24 +4,32 @@ plugins {
 }
 
 android {
-    compileSdkVersion(30)
+    namespace = "com.gu.toolargetool.sample"
+    compileSdk = 33
     defaultConfig {
-        applicationId("com.gu.toolargetool.sample")
-        minSdkVersion(15)
-        targetSdkVersion(30)
-        versionCode(1)
-        versionName("1.0")
-        testInstrumentationRunner("androidx.test.runner.AndroidJUnitRunner")
+        applicationId = "com.gu.toolargetool.sample"
+        versionCode = 2
+        versionName = "1.1"
```

Comment:
Don't need to increment them. This is sample app for the library, so there's no deployment to store where versions need incrementing.